Files/Views/Settings.xaml.cs

```diff
@@ -29,6 +30,8 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
             var currentView = SystemNavigationManager.GetForCurrentView();
             currentView.BackRequested += OnBackRequested;
 
+            Window.Current.CoreWindow.Dispatcher.AcceleratorKeyActivated += CoreDispatcher_AcceleratorKeyActivated;
```

Comment:
Could that be done in MainPage.xaml?

Reply:
It’s technically possible. But why ?
This is a behavior of the Settings page, It should be in there.
In addition, we can manage other shortcuts later, to use the settings entirely from the keyboard.